administrator/components/com_media/resources/scripts/components/modals/create-folder-modal.vue

```diff
@@ -23,13 +23,30 @@
     export default {
         name: 'create-folder-modal',
         directives: { focus: focus },
+        ready: function() {
+            /*add event listener on keydown*/
+            document.addEventListener("keydown", (e) => {
+                /*checks if the model is shown and enter(keycode 13) is pressed*/
+                if($store.state.showCreateFolderModal && e.keyCode == 13){
+                    save();
+                    close();
+                }
+            });
```

Comment:
Do we need this additional event listener? @submit.prevent="save" should do the trick without it.